runtime/src/bank.rs

```diff
@@ -5980,6 +5980,11 @@ impl Bank {
             .is_active(&feature_set::verify_tx_signatures_len::id())
     }
 
+    pub fn libsecp256k1_0_5_upgrade_enabled(&self) -> bool {
```

Comment:
Where did this change come from?

Reply:
Just some old code that I forgot to remove.